configs/envoy_service_to_service_v2.template.yaml

```diff
@@ -0,0 +1,429 @@
+    type: STRICT_DNS
+    lb_policy: RANDOM
+    hosts:
+    {% for server in value['hosts'] -%}
+    - socket_address:
+        protocol: {{ server['protocol'] }}
+        port_value: {{ server['port_value'] }}
+        address: {{ server['address'] }}
+    {% if not loop.last -%}{% endif -%}
+    {% endfor -%}
+  {% endfor %}
+  - name: main_website
+    connect_timeout: 0.25s
+    type: LOGICAL_DNS
+    # Comment out the following line to test on v6 networks
+    dns_lookup_family: V4_ONLY
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        address: main_website.com
+        port_value: 443
+    tls_context: { sni: www.main_website.com }
+  - name: local_service
+    connect_timeout: 0.25s
+    type: STATIC
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 8080
+    circuit_breakers:
+      thresholds:
+        max_pending_requests: 30
+        max_connections: 100
+  - name: local_service_grpc
+    connect_timeout: 0.25s
+    type: STATIC
+    lb_policy: ROUND_ROBIN
+    http2_protocol_options: {}
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 8081
+    circuit_breakers:
+      thresholds:
+        max_requests: 200
+  - name: rds
+    connect_timeout: 0.25s
+    type: STRICT_DNS
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: rds.yourcompany.net
+        port_value: 80
+  - name: statsd
+    connect_timeout: 0.25s
+    type: STATIC
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: 127.0.0.1
+        port_value: 8125
+  - name: lightstep_saas
+    connect_timeout: 1s
+    type: LOGICAL_DNS
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: collector-grpc.lightstep.com
+        port_value: 443
+    http2_protocol_options:
+      max_concurrent_streams: 100
+    tls_context:
+      common_tls_context:
+        validation_context:
+          trusted_ca:
+            filename: certs/cacert.pem
+          verify_subject_alt_name:
+          - collector-grpc.lightstep.com
+  - name: cds_cluster
+    connect_timeout: 0.25s
+    type: STRICT_DNS  
+    lb_policy: ROUND_ROBIN
+    hosts:
+    - socket_address:
+        protocol: TCP
+        address: cds.yourcompany.net
+        port_value: 80
+flags_path: /etc/envoy/flags
+stats_sinks:
+  - name: envoy.statsd
+    config:
+      tcp_cluster_name: statsd
+tracing:
+  http:
+    name: envoy.lightstep
+    config:
+      access_token_file: /etc/envoy/lightstep_access_token
+      collector_cluster: lightstep_saas
+rate_limit_service:
+  grpc_service:
+    envoy_grpc:
+      cluster_name: ratelimit
+admin:
+  access_log_path: /var/log/envoy/admin_access.log
+  address:
+    socket_address:
+      protocol: TCP
+      address: 0.0.0.0
+      port_value: 9901
```

Comment:
missing a cluster_manager field, and a dynamic_resources field with a cds_config https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/bootstrap/v2/bootstrap.proto#config-bootstrap-v2-bootstrap-dynamicresources

Reply:
couldn't find related configurations in old template. probably need some help here.

Reply:
@srikailash sorry, disregard the cluster manager portion of that comment. You are right, that was not there before